blobmodel/plotting.py

```diff
@@ -0,0 +1,63 @@
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as np
+from tqdm import tqdm
+from blobmodel import Model
+
+
+def show_model(
+    model: Model,
+    interval: int = 100,
+    save: bool = False,
+    gif_name: str = "2d_blobs.gif",
+    fps: int = 10,
+) -> None:
+    """
+        show animation of Model
+
+        Parameters
+        ----------
+        interval: int, optional
+            time interval between frames in ms
+        save: bool, optional
+            if True save animation as gif
+        gif_name: str, optional
+            set name for gif
+        fps: int, optional
+            set fps for gif
+        """
+    fig = plt.figure()
+    ax = fig.add_subplot(111)
+    div = make_axes_locatable(ax)
+    cax = div.append_axes("right", "5%", "5%")
+
+    frames = []
+
+    __xx, __yy = np.meshgrid(model.x, model.y)
+
+    for t in tqdm(model.t, desc="Creating frames for animation"):
+        curVals = np.zeros(shape=(model.Ny, model.Nx))
+        for b in model.get_blobs():
+            curVals += b.discretize_blob(
+                x=__xx, y=__yy, t=t, periodic_y=model.periodic_y, Ly=model.Ly
+            )
+        frames.append(curVals)
```

Comment:
I hadn't realized this before, but I think it is a bit strange that the plotting function actually generates the data. You would usually have a function that generate data (integrate method in model) and then you might plot the data separately. i.e. The lines 39-45 are unnecessarily repeated code imo

Reply:
I think this plotting method should either generate the data by calling model.integrate(), or take the simulation data as an argument.